examples/example-gateway/endpoints/baz/baz_simpleservice_method_call.go

```diff
@@ -0,0 +1,64 @@
+// TODO: (lu) to be generated
+
+package baz
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/uber-go/zap"
+	"github.com/uber/zanzibar/examples/example-gateway/build/clients"
+	zanzibar "github.com/uber/zanzibar/runtime"
+
+	clientTypeBaz "github.com/uber/zanzibar/examples/example-gateway/build/gen-code/github.com/uber/zanzibar/clients/baz/baz"
+	"github.com/uber/zanzibar/examples/example-gateway/build/gen-code/github.com/uber/zanzibar/endpoints/baz/baz"
+)
+
+// HandleCallRequest handles "/baz/call-path".
+func HandleCallRequest(
```

Comment:
I just landed a change to the semantics of "custom" endpoints. you'll need to rebase on master and change this interface.